main.py

```diff
@@ -1,7 +1,53 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genre_to_create = ["Western", "Action", "Dramma"]
+
+    actors_to_create = [("George", "Klooney"),
+                        ("Kianu", "Reaves"),
```

Comment:
The names 'George Klooney' and 'Kianu Reaves' are likely typos. They are later corrected to 'George Clooney' and 'Keanu Reeves', respectively. Consider correcting them during the initial creation to avoid the need for updates.